examples/cp-all-in-one/docker-compose.yml

```diff
@@ -11,16 +11,23 @@ services:
       ZOOKEEPER_TICK_TIME: 2000
 
   broker:
-    image: confluentinc/cp-kafka
+    image: confluentinc/cp-enterprise-kafka
     hostname: broker
     depends_on:
       - zookeeper
     ports:
-      - '9092'
+      - "9092"
```

Comment:
I realize you didn't introduce this. Should this be '9092:9092' given that zookeeper is open?